.github/workflows/e2e-upgrade.yaml

```diff
@@ -197,9 +197,39 @@ jobs:
       chain-binary: simd
       chain-a-tag: v8.3.2
       chain-b-tag: v8.3.2
-      chain-upgrade-tag: pr-6791 # TODO: Update tag to v9.0.0 once it is (pre)released
+      chain-upgrade-tag: pr-6806 # TODO: Update tag to v9.0.0 once it is (pre)released
       upgrade-plan-name: "v9"
       test-entry-point:  "TestUpgradeTestSuite"
       test: "TestV8ToV9ChainUpgrade_Localhost"
       upload-logs: true
-      relayer-type: hermes
\ No newline at end of file
+      relayer-type: hermes
+
+  upgrade-v9-channel-upgrades-hermes:
+    uses: ./.github/workflows/e2e-test-workflow-call.yml
+    with:
+      chain-image: ghcr.io/cosmos/ibc-go-simd
+      chain-binary: simd
+      chain-a-tag: v8.3.2
+      chain-b-tag: pr-6806 # TODO: Update tag to v9.0.0 once it is (pre)released
+      chain-upgrade-tag: pr-6806 # TODO: Update tag to v9.0.0 once it is (pre)released
+      upgrade-plan-name: "v9"
+      test-entry-point:  "TestUpgradeTestSuite"
+      test: "TestV8ToV9ChainUpgrade_ChannelUpgrade"
+      upload-logs: true
+      relayer-type: hermes
+
+  upgrade-v9-channel-upgrades-rly:
+    uses: ./.github/workflows/e2e-test-workflow-call.yml
+    with:
+      chain-image: ghcr.io/cosmos/ibc-go-simd
+      chain-binary: simd
+      chain-a-tag: v8.3.2
+      chain-b-tag: pr-6806 # TODO: Update tag to v9.0.0 once it is (pre)released
+      chain-upgrade-tag: pr-6806 # TODO: Update tag to v9.0.0 once it is (pre)released
```

Comment:
hm actually these can probably just use `main`. Since we aren't making changes to the simapp or anything.

Reply:
True, changed to that :)